content/smyja/how-to-use-graphql-with-django.md

```diff
@@ -0,0 +1,288 @@
+---
+Title: "How to Use GraphQL With Django"
+Description: "A guide on how to use GraphQL with Django."
+DatePublished: "2022-06-05"
+Categories:
+  - "web-development"
+  - "computer-science"
+Tags:
+  - "APIs"
+  - "Django"
+  - "Python"
```

Comment:
```suggestion
Categories:
  - "python"
  - "web-development"
  - "computer-science"
Tags:
  - "APIs"
  - "Django"
```

Looks like "Python" is not in [tags.md](https://github.com/Codecademy/ugc/blob/main/documentation/tags.md). However, it _is_ in [categories.md](https://github.com/Codecademy/ugc/blob/main/documentation/categories.md) under language slugs (but in lowercase).

This change should also hopefully resolve the failing GitHub check we're getting.